README.md

```diff
@@ -10,7 +10,15 @@ Information about our deployments can be found on our [Google Drive document](ht
 
 ### The demoservice app
 
-#### The easy way (Docker - Untested)
+#### Docker Compose
+
+A close version to the production set up can be started using docker compose. All services are defined under docker-compose.yml in the project root.
+
+``` bash
+docker-compose up
+```
+
+#### Docker - Untested
```

Comment:
Unless there's a good reason, could you please keep this clean by removing this section entirely, and also removing the associated `bin/start_docker_dev`?